test_util/test_aws_cf.py

```diff
@@ -0,0 +1,140 @@
+    options.pytest_dir = os.getenv('DCOS_PYTEST_DIR', '/opt/mesosphere/active/dcos-integration-test')
+    options.pytest_cmd = os.getenv('DCOS_PYTEST_CMD', 'py.test -vv '+options.ci_flags)
+    return options
+
+
+def main():
+    options = check_environment()
+
+    random_identifier = ''.join(random.choice(string.ascii_uppercase + string.digits) for _ in range(10))
+    unique_cluster_id = "CF-integration-test-{}".format(random_identifier)
+    log.info("Spinning up AWS CloudFormation with ID: {}".format(unique_cluster_id))
+    bw = test_util.provide_aws.BotoWrapper(
+            region=DEFAULT_AWS_REGION,
+            aws_access_key_id=options.aws_access_key_id,
+            aws_secret_access_key=options.aws_secret_access_key)
+    # TODO(mellenburg): use randomly generated keys this key is delivered by CI or user
+    ssh_key_path = 'default_ssh_key'
+    cf = test_util.provide_aws.DcosCfSimple.create(
+        stack_name=unique_cluster_id,
+        template_url=options.template_url,
+        private_agents=2,
+        public_agents=1,
+        admin_location='0.0.0.0/0',
+        key_pair_name='default',
+        boto_wrapper=bw)
+    cf.wait_for_stack_creation()
+
+    log.info("Exporting ssh_key to disk as")
+    # key must be chmod 600 for test_runner to use
+    os.chmod(ssh_key_path, stat.S_IREAD | stat.S_IWRITE)
+
+    # Create custom SSH Runnner to help orchestrate the test
+    ssh_user = 'core'
+    remote_dir = '/home/core'
+
+    master_ips = cf.get_master_ips()
+    public_agent_ips = cf.get_public_agent_ips()
+    private_agent_ips = cf.get_private_agent_ips()
+    test_host = master_ips[0][1]
+    log.info('Running integration test from: '+test_host)
+    master_list = [i[0] for i in master_ips]
+    log.info('Master private IPs: '+repr(master_list))
+    agent_list = [i[0] for i in private_agent_ips]
+    log.info('Private agent private IPs: '+repr(agent_list))
+    public_agent_list = [i[0] for i in public_agent_ips]
+    log.info('Public agent private IPs: '+repr(public_agent_list))
+
+    with closing(SSHTunnel(ssh_user, ssh_key_path, test_host)) as test_host_tunnel:
+        # Allow docker use w/o sudo
+        result = test_util.test_runner.integration_test(
+                tunnel=test_host_tunnel,
+                test_dir=remote_dir,
+                region=DEFAULT_AWS_REGION,
+                dcos_dns=master_list[0],
+                master_list=master_list,
+                agent_list=agent_list,
+                public_agent_list=public_agent_list,
+                provider='aws',
+                test_dns_search=False,
+                aws_access_key_id=options.aws_access_key_id,
+                aws_secret_access_key=options.aws_secret_access_key,
+                add_env=options.add_env,
+                pytest_dir=options.pytest_dir,
+                pytest_cmd=options.pytest_cmd)
+    if result == 0:
+        log.info("Test successsful! Deleting CloudFormation and generated keypair...")
```

Comment:
This talks about deleting the keypair but doesn't actually have any code to do so. Update the message to reflect the current code?